cookbook/scripts/helper.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Let's call the file something more descriptive... maybe `tests_helper.sh` or something...

Reply:
ci_helper look good because we also use it  for release